docset/winserver2022-ps/smbshare/New-SmbShare.md

```diff
@@ -412,6 +412,9 @@ Accept wildcard characters: False
 Shows what would happen if the cmdlet runs.
 The cmdlet is not run.
 
+> [!NOTE]
```

Comment:
Are we allowing Note tags in cmdlet articles now? maybe I don't know the last news about this...

Reply:
@get-itips - Thank you, Andres, missed it. No, Note tags are not allowed.